components/card/Card.tsx

```diff
@@ -4,11 +4,13 @@ import { Title, Wrapper } from './Card.styles';
 
 interface CardProps {
   title: string;
+  width?: number;
```

Comment:
Because reddit is a test project, this is fine to solve your problems. But in a real project, we should never pass layout props to a similar (Card) component. The layout should be defined in the page component. Think of the actual reddit.com. The card of Trending/Card is inside a Sidebar component. The Sidebar component exists in the Home page. It's either the Home page that defines the width of the Sidebar component (e.g. 200px), or if we want the Sidebar component to exist in multiple screens and always have the same width, then we can define the width inside the Sidebar component. The Card component is very low in the layout tree (HomePage -> Sidebar -> Card) to be responsible for the layout. Ask me again tomorrow if this is not clear :)

Reply:
Also, you need the width for a specific use case (Home page). I would set the width and height as "auto" by default and only if the user passes a value only then I would overwrite auto. There's no need to set a width/height in the AddCommunity page too.